ui/app/AppLayouts/Chat/popups/PinnedMessagesPopup.qml

```diff
@@ -53,6 +53,25 @@ StatusDialog {
             delegate: Item {
                 id: messageDelegate
 
+                property bool isUserAdded: false
+
+                function updateUserAdded() {
+                    const contactDetails = Utils.getContactDetailsAsJson(model.senderId, false)
+                    isUserAdded = contactDetails && contactDetails.isAdded
+                }
+
+                Connections {
+                    target: root.store.contactsStore.myContactsModel
+
+                    function onItemChanged(pubKey) {
+                        if (model.senderId === pubKey) {
```

Comment:
Are you sure the `model` is still accessible here inside the `Connections`?

Reply:
that part was working, but i'm currently changing approach because i need chatId, not senderId